models/public/retinaface-resnet50/retinaface-resnet50.md

```diff
@@ -0,0 +1,89 @@
+# RetinaFace-ResNet50
+
+## Use Case and High-Level Description
+RetinaFace-R50 is a medium size model with ResNet50 backbone for Face Localisation. It can output face bounding boxes and five facial landmarks in a single forward pass. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/deepinsight/insightface/tree/master/RetinaFace)
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP ([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/)) | 87.30%                  |
+| GFLOPs                                                        | 100.8478                |
+| MParams                                                       | 29.4276                 |
+| Source framework                                              | MXNet\*                 |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. All numbers were evaluated by taking into account only faces bigger than
+64 x 64 pixels.
+
+Accuracy validation approach different from described in the original repo.
+For details about original WIDER results please see [https://github.com/deepinsight/insightface/tree/master/RetinaFace]()
+
+## Input
+
+### Original model:
+Image, name: `data`,  shape: `1,3,640,640`, format: `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+
+### Converted model:
+Image, name: `data`,  shape: `1,3,640,640`, format: `B,C,H,W`, where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+model outputs is floating points vectors:
+1. name: `face_rpn_cls_prob_reshape_stride32`, shape: `[1x4x20x20]` - Detection scores for Feature Pyramid Network (FPN) level with stride 32
+2. name: `face_rpn_bbox_stride32`,  shape: `[1x8x20x20]` -  Detection boxes for Feature Pyramid Network (FPN) level with stride 32
+3. name: `face_rpn_landmark_pred_stride32`, shape: `[1x20x20x20]` - Facial Landmarks for Feture Pyramid Network (FPN) level with stride 32
```

Comment:
Please, add format of output and description if necessary. [This](https://github.com/opencv/open_model_zoo/blob/develop/models/public/yolo-v3-tf/yolo-v3-tf.md) and [this](https://github.com/opencv/open_model_zoo/blob/develop/models/public/ssd_mobilenet_v1_coco/ssd_mobilenet_v1_coco.md) could help you.

Reply:
it is very difficult and it does not help me....

Reply:
I tried to describe them, please look